api/tests/lib/seed_orgs_and_users.py

```diff
@@ -500,3 +510,52 @@ def handle_static_application_forms(application: Application, competition: Compe
         )
 
         validate_application_form(application_form, ApplicationAction.START)
+
+
+def _build_legacy_users_for_orgs(
+    orgs: list[Organization],
+    inviter: User,
+    user_scenarios: list[str],
```

Comment:
I'd leave the user_scenarios out of this, that is meant more for logging what users we created that you can login as, these wouldn't be those kinds of users.